src/component/marker/MarkerView.ts

```diff
@@ -87,6 +90,21 @@ abstract class MarkerView extends ComponentView {
             }
         });
     }
+    updateZ(ecModel: GlobalModel, markerGroupMap: HashMap<MarkerDraw>): void {
+        ecModel.eachSeries(seriesModel => {
+
+            const markerModel = MarkerModel.getMarkerModelFromSeries(
+                seriesModel,
+                this.type as 'markPoint' | 'markLine' | 'markArea'
+            );
+
+            if (markerModel) {
+                const { z, zlevel } = retrieveZInfo(markerModel);
+                traverseUpdateZ(markerGroupMap.get(seriesModel.id)?.group, z, zlevel);
```

Comment:
Sorry my code misled you yesterday.
Logically the optional chain (the question mark `?`) here is not reasonable.

`traverseUpdateZ` does not perform null checking internally.

Therefore, here we either remove the question mark (assert not null)
or perform a null checking for better robustness.